File: script/python/pyprogramminginth/CharCheck.py
a = str(input("Enter any string in single line(don't space):"))
upper = 0
lower = 0
for char in a:
    if char.isupper():
        upper += 1
    elif char.islower():
        lower += 1

if upper != 0 and lower == 0:
    print("All Capital Letter")
elif upper == 0 and lower != 0:
    print("All Small Letter")
else:
    print("Mix")

File: compile_lang/C++/cppprogramminginth/charcheck/src/charcheck.cpp
#include <iostream>

int main() {
    int c, big = 0, small = 0;

    while ((c = getchar()) != EOF && c != '\n') {
        if ('A' <= c && c <= 'Z') big = 1;
        if ('a' <= c && c <= 'z') small = 1;
        if (big + small == 2) break;
    }
    
    std::cout << ((big && small)? "Mix": (big)? "All Capital Letter": "All Small Letter") << std::endl;

    return 0;
}